### PR TITLE
CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by EmilienLeroy**
Added a Contributing.md with those sections :

* Code of Conduct
* How to contribute
    * Install the project
    * Project Structure
    * Npm script reference (missing function description)

I think it's a good begin for a contributing.md 😋

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Contributing to Hex-engine
+
+First off, thanks for taking the time to contribute !
+
+The following is a set of guidelines for contributing to Hex-engine. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
The following is a set of guidelines for contributing to Hex Engine. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+* [Code of Conduct](#code-of-conduct)
+* [How to contribute](#how-to-contribute)
+    * [Install the project](#install-the-project)
+    * [Project Structure](#project-structure)
+    * [npm script reference](#npm-script-reference)
+
+## Code of Conduct
+
+By participating to hex-engine, you pledge to uphold the standards listed in the Contributor Covenant Code of Conduct, as found here: https://www.contributor-covenant.org/version/2/0/code_of_conduct
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
By participating to Hex Engine, you pledge to uphold the standards listed in the Contributor Covenant Code of Conduct, as found here: https://www.contributor-covenant.org/version/2/0/code_of_conduct
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## How to contribute
+
+Hex-engine greatly appreciates PRs, but open an issue first to check if the change you want to make are in line with the project vision.
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
Hex Engine greatly appreciates pull requests, but open an issue first to check if the change you want to make are in line with the project vision.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+# Contributing to Hex-engine
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
# Contributing to Hex Engine
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+Hex-engine greatly appreciates PRs, but open an issue first to check if the change you want to make are in line with the project vision.
+
+### Install the project
+
+In first you need to fork and clone this repository.
+```
+git clone https://github.com/<your-gh-user-name>/hex-engine.git
+```
+
+After install the dependencies. Hex-engine use *yarn* to manage dependency. If you don't have *yarn* just install it with this command :
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
Afterwards install the dependencies. Hex Engine use *yarn* to manage dependencies. If you don't have *yarn* just install it with this command : 
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+```
+npm install -g yarn 
+```
+
+And you can install the dependencies with this command line : 
+
+```
+yarn install
+```
+Now you can run the hex-engine project with the following command :
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
Now you can run the Hex Engine project with the following command : 
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+```
+Now you can run the hex-engine project with the following command : 
+
+```
+yarn start
+```
+
+### Project Structure
+
+This project is separate in multiple packages which you can find into the *packages* folder.
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
This project is separated in multiple packages which you can find in the *packages* folder.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+| Package                      | Description                             |
+| ---------------------------- | --------------------------------------- |
+| `2d`                         | Provide primitives for making a 2D game.|
+| `core`                       | Core Entity and Component system.       |
+| `create`                     | Cli to create boilerplate project.      |
+| `game`                       | Example of use case with hex-engine.    |
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
| `game`                       | Example of use case with Hex Engine.    |
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+| `inspectore`                 | Integrate the inspectore to debug hex game. |
+| `scripts`                    | Cli for hex-engine.                     |
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
| `scripts`                    | CLI for Hex Engine.                     |
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+This project is separate in multiple packages which you can find into the *packages* folder.
+
+| Package                      | Description                             |
+| ---------------------------- | --------------------------------------- |
+| `2d`                         | Provide primitives for making a 2D game.|
+| `core`                       | Core Entity and Component system.       |
+| `create`                     | Cli to create boilerplate project.      |
+| `game`                       | Example of use case with hex-engine.    |
+| `inspectore`                 | Integrate the inspectore to debug hex game. |
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
| `inspector`                 | Integrate the inspector to debug Hex Engine projects. |
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+| `scripts`                    | Cli for hex-engine.                     |
+| `website`                    | Hex-engine website.                     |
```
Comment (arya-s) [inline CONTRIBUTING.md]:
```suggestion
| `website`                    | Hex Engine website.                     |
```

**Comment by suchipi** [comment]
Thank you for getting this ball rolling! I really appreciate it!

**Pull request merged**